test/server.cpp

```diff
@@ -735,7 +735,7 @@ TEST_F(LibraryServerTest, catalog_search_by_phrase)
   EXPECT_EQ(maskVariableOPDSFeedData(r->body),
     OPDS_FEED_TAG
     "  <id>12345678-90ab-cdef-1234-567890abcdef</id>\n"
-    "  <title>Filtered zims (q=&quot;ray charles&quot;)</title>\n"
+    "  <title>Filtered zims (?q=&quot;ray charles&quot;)</title>\n"
```

Comment:
Do we really want this change ?

(But not sure it is really important, we don't use the feed title)